rllib/evaluation/worker_set.py

```diff
@@ -74,9 +74,10 @@ def remote_workers(self):
 
     def sync_weights(self):
         """Syncs weights of remote workers with the local worker."""
-        weights = ray.put(self.local_worker().get_weights())
-        for e in self.remote_workers():
-            e.set_weights.remote(weights)
+        if self.remote_workers():
```

Comment:
Bug fix to avoid needing to call Ray APIs with num_workers=0.

Reply:
+1